modules/tinymce/src/plugins/preview/main/ts/core/IframeContent.ts

```diff
@@ -17,15 +17,15 @@ const getPreviewHtml = (editor: Editor) => {
 
   headHtml += '<base href="' + encode(editor.documentBaseURI.getURI()) + '">';
 
-  if (contentStyle) {
-    headHtml += '<style type="text/css">' + contentStyle + '</style>';
-  }
-
   const cors = Settings.shouldUseContentCssCors(editor) ? ' crossorigin="anonymous"' : '';
   Tools.each(editor.contentCSS, (url) => {
     headHtml += '<link type="text/css" rel="stylesheet" href="' + encode(editor.documentBaseURI.toAbsolute(url)) + '"' + cors + '>';
   });
 
+  if (contentStyle) {
```

Comment:
Can we add a test for this?

Reply:
Test already exists.. what I've done here is simply altering the order.

Reply:
Then why isn't the test failing? as there are no changes to tests in the preview plugin.

Reply:
It may never existed @lnewson. I'll check it out.

Reply:
Hi @lnewson , I've just added a new test on PreviewContentStyleTest.ts, please have a look on that and let me know if it fulfils the requirements.